bitnami/mariadb-galera/values-production.yaml

```diff
@@ -311,7 +308,9 @@ mariadbConfiguration: |-
   wsrep_cluster_address=gcomm://
   wsrep_cluster_name=galera
   wsrep_sst_auth="root:"
-  innodb-flush-log-at-trx-commit=2
+  # Not fully ACID compliant, up to 1sec transaction loss in the event of total cluster failure (across both regions)
+  # Enabled for performance per https://mariadb.com/kb/en/configuring-mariadb-galera-cluster/
+  innodb_flush_log_at_trx_commit=0
```

Comment:
Before, we were setting `innodb_flush_log_at_trx_commit=0` to the `[mysqld]` section,  `innodb_flush_log_at_trx_commit=1` to the `[mariadb]` section and `innodb_flush_log_at_trx_commit=2` to the `[galera]` section.

But now we are setting `innodb_flush_log_at_trx_commit=1` to `[mariadb]`, `innodb_flush_log_at_trx_commit=0` to `[galera]` and none to MySQL (should be =0).

It looks like a typo was made here as one is missing. Could you fix that? I've also seen that in the file below.